doc/guides/test/sanitycheck.rst

```diff
@@ -1,33 +1,34 @@
 
 .. _sanitycheck_script:
 
-Zephyr Sanity Tests
-###################
+Sanity Tests
+#############
```

Comment:
There's more documentation in the docstring in the sanitycheck python script that's not included here.  Should it be?